pkgs/development/python-modules/azure-core/default.nix

```
@@ -51,7 +51,25 @@ buildPythonPackage rec {
 
   pytestFlagsArray = [ "tests/" ];
   # disable tests which touch network
-  disabledTests = [ "aiohttp" "multipart_send" "response" "request" "timeout" ];
+  disabledTests = [
+    "aiohttp"
+    "multipart_send"
+    "response"
+    "request"
+    "timeout"
+  # disable tests failing on some darwin machines with errors:
+  # azure.core.polling.base_polling.BadStatus: Invalid return status 403 for 'GET' operation
+  # azure.core.exceptions.HttpResponseError: Operation returned an invalid status 'Forbidden'
+  ] ++ lib.optional stdenv.isDarwin [
+    "test_patch_location_polling_fail"
+    "test_patch_location_polling_fail"
+    "test_patch_operation_location_polling_fail"
+    "test_patch_operation_location_polling_fail"
+    "test_put_location_polling_fail"
+    "test_put_location_polling_fail"
+    "test_put_operation_location_polling_fail"
+    "test_put_operation_location_polling_fail"
+  ];
```

This gets passed to pytest's `-k 'not ...`, so you just need a common substring for all of them
```suggestion
  ] ++ lib.optional stdenv.isDarwin [
    "location_polling"
  ];
```

---

Nice, didn't know that.

Done.